meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -534,6 +543,72 @@ PREFERRED_PROVIDER_u-boot-default-script:stm32mp1common = "u-boot-base-scr"
 PREFERRED_PROVIDER_u-boot-default-script:sota:stm32mp1common = "u-boot-ostree-scr-fit"
 # Remove standard ST BSP from SDK
 TOOLCHAIN_HOST_TASK:remove:stm32mp1common = "${ST_TOOLS_FOR_SDK} ${ST_DEPENDENCIES_BUILD_FOR_SDK}"
+STM32_BOOTIMAGE_SUFFIX ??= ""
+MFGTOOL_FLASH_IMAGE ?= "${IMAGE_BASENAME}-${MACHINE}"
```

Comment:
This will also affect the mfgtool-files recipe for imx, I think it is better for now to just set LMP_FLASHLAYOUT_IMAGE directly to ${IMAGE_BASENAME}-${MACHINE} and avoid setting it up.

Reply:
that was used before in the recipe `meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-support/flashlayouts-stm32mp1/flashlayouts-stm32mp1.bb`. Let's keep it for now, and address that as improvement in future?

Reply:
I think one of the issues is because it affects IMX as well so we need to STM specific. Using the `stm32mp1common` machine override fixes that part.
```
MFGTOOL_FLASH_IMAGE:stm32mp1common ?= "${IMAGE_BASENAME}-${MACHINE}"
```

Reply:
Yeah, before it was isolated in the recipe, now it is global and will also affect imx, which is why it is better to make it stm32 specific.

Reply:
Either what @quaresmajose suggested or a new variable would be fine.